executor/executor.go

```diff
@@ -233,6 +234,7 @@ func Run(path string, env []string, emitter screwdriver.Emitter, build screwdriv
 	setupCommands := []string{
 		"set -e",
 		"export PATH=$PATH:/opt/sd",
+		". " + baseEnvFile,
```

Comment:
what happened if this cmd failed? e.g. user set some really weird env.

Reply:
It will output something like this, just same way it will happen in a shell
```
bash-3.2$ . /tmp/export && echo "bad"
bash: export: `a.b=WJFDS': not a valid identifier
```

Reply:
Moved sourcing to a step so that step is updated